src/css/components/Navigation.css

```diff
@@ -1,18 +1,22 @@
+.navbar {
+  position: absolute;
+  bottom: 0px;
```

Comment:
When you have `0` for the value, you can always omit the unit (0 of any unit is just...0). So this can change to `bottom: 0;`